jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb

```diff
@@ -0,0 +1,379 @@
+   "source": [
+    "import ai.djl.Application;\n",
+    "import ai.djl.MalformedModelException;\n",
+    "import ai.djl.ModelException;\n",
+    "import ai.djl.inference.Predictor;\n",
+    "import ai.djl.modality.Classifications;\n",
+    "import ai.djl.modality.cv.*;\n",
+    "import ai.djl.modality.cv.output.*;\n",
+    "import ai.djl.modality.cv.transform.*;\n",
+    "import ai.djl.modality.cv.translator.ImageClassificationTranslator;\n",
+    "import ai.djl.modality.cv.util.NDImageUtils;\n",
+    "import ai.djl.ndarray.*;\n",
+    "import ai.djl.ndarray.types.Shape;\n",
+    "import ai.djl.repository.zoo.*;\n",
+    "import ai.djl.translate.*;\n",
+    "\n",
+    "import java.io.IOException;\n",
+    "import java.nio.file.*;\n",
+    "import java.util.*;"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Face Detection model\n",
+    "\n",
+    "Now we can start working on the first model. The model can do face detection and require some additional processing before we feed into it:\n",
+    "\n",
+    "- Resize: Shrink the image with a certain ratio to feed in\n",
+    "- Normalize the image with a scale\n",
+    "\n",
+    "Fortunatly, DJL offers a `Translator` interface that can help you with these processing. The rough Translator architecture looks like below:\n",
+    "\n",
+    "![](https://github.com/awslabs/djl/blob/master/examples/docs/img/workFlow.png?raw=true)\n",
+    "\n",
+    "In the following sections, we will implement a `FaceTranslator` class to do the work.\n",
+    "\n",
+    "### Preprocessing\n",
+    "\n",
+    "In this stage, we will load an image and do some preprocessing work to it. Let's load the image first and take a look at it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "String url = \"https://raw.githubusercontent.com/PaddlePaddle/PaddleHub/release/v1.5/demo/mask_detection/python/images/mask.jpg\";\n",
+    "Image img = ImageFactory.getInstance().fromUrl(url);\n",
+    "img.getWrappedImage();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then, let's try to apply some transformation to it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "NDList processImageInput(NDManager manager, Image input, float shrink) {\n",
+    "            NDArray array = input.toNDArray(manager);\n",
+    "            Shape shape = array.getShape();\n",
+    "            array = NDImageUtils.resize(\n",
+    "                            array, (int) (shape.get(1) * shrink), (int) (shape.get(0) * shrink));\n",
+    "            array = array.transpose(2, 0, 1).flip(0); // HWC -> CHW BGR -> RGB\n",
+    "            NDArray mean = manager.create(new float[] {104f, 117f, 123f}, new Shape(3, 1, 1));\n",
+    "            array = array.sub(mean).mul(0.007843f); // normalization\n",
```

Comment:
maybe use NDimageUtils.normalize here

Reply:
normailize applies std, where this one use multiplication instead